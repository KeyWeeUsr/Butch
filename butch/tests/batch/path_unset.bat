path ;
echo %errorlevel%
path
echo %errorlevel%
echo %PATH%
echo %errorlevel%
