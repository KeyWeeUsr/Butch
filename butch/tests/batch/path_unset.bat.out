0
PATH=(null)
0

0
